src/prpy/planning/base.py

```diff
@@ -298,6 +318,23 @@ def plan(self, method, args, kw_args):
                     with Timer() as timer:
                         output = planner_method(*args, **kw_args)
 
+                    if not is_sequence_deterministic:
+                        # TODO: It is overly conservative to set _ENDPOINT,
+                        # e.g. for PlanToConfiguration. Unfortunately, there is
+                        # no easy way to detect this special case.
+                        SetTrajectoryTags(output, {
+                            Tags.DETERMINISTIC_TRAJECTORY: False,
+                            Tags.DETERMINISTIC_ENDPOINT: False,
```

Comment:
Can we check the existing `Tags.DETERMINISTIC_ENDPOINT` of `output`? `PlanToConfiguration` should set this to `True` before returning a trajectory.